src/librustdoc/html/static/main.js

```diff
@@ -142,8 +142,8 @@ function getSearchElement() {
     var TY_PRIMITIVE = itemTypes.indexOf("primitive");
     var TY_KEYWORD = itemTypes.indexOf("keyword");
 
-    onEachLazy(document.getElementsByClassName("js-only"), function(e) {
-        removeClass(e, "js-only");
+    onEachLazy([getSearchInput()], function(e) {
```

Comment:
Also: no need anymore for `onEachLazy`: just grab the output of `getSearchOutput` and use it:

```js
var e = getSearchInput();
if (e) {
    e.removeAttribute('disabled');
}
```